lesson_02/math/main.go

```diff
@@ -0,0 +1,31 @@
+package math
+
+import "fmt"
+
+func Fibonacci(n int) {
```

Comment:
I would rather use `Fibonacci` inside `FmtFibonacci`, because if I needed just the Fibonacci sequence, I would prefer a slice of numbers without printing them out.